# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../strace/arch-sparc.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Oh well, the usual non-mainstream arch colateral breakage we are used to, ...

  - Rene Rebe <rene@exactcode.de>

--- strace-4.5.20/linux/sparc/syscall.h.vanilla	2010-07-20 15:47:51.781063554 +0000
+++ strace-4.5.20/linux/sparc/syscall.h	2010-07-20 15:50:14.690047947 +0000
@@ -214,6 +214,7 @@
 int	sys_move_pages(), sys_getcpu();
 int	sys_epoll_pwait();
 int	sys_signalfd(), sys_timerfd(), sys_eventfd();
+int	sys_mbind(), sys_get_mempolicy(), sys_set_mempolicy(), sys_utimensat(), sys_timerfd_create(), sys_fallocate(), sys_timerfd_settime(), sys_timerfd_gettime(), sys_signalfd4(), sys_eventfd2(); sys_epoll_create1(), sys_dup3(), sys_pipe2(), sys_inotify_init1();
 
 #  define SYS_socket_subcall	353
 #define SYS_sub_socket		(SYS_socket_subcall + 1)
